app/src/components/navbar/NavbarComponent.jsx

```diff
@@ -77,6 +77,15 @@ function NavbarComponent() {
                 <Link to="/blogs/Durgamandir" className="font-15 text-brown font-regular dropdown-item">
                   Durga Mata Mandir
                 </Link>
+                <Link to="/blogs/VaranasiItinary" className="font-15 text-brown font-regular dropdown-item">
+                  Varanasi Itinary
```

Comment:
Hard coding name is bad practice. Try picking it from constants file. Create one if it do not exists.

Reply:
ok